python/cugraph/core/k_core.py

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) 2019, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from cugraph.core import k_core_wrapper
+from cugraph.structure.graph import Graph
+
+
+def k_core(G,
+           k=None,
+           core_number=None):
+    """
+    Compute the core numbers for the nodes of the graph G.
```

Comment:
Better description here.  In fact, this is misleading.  k_core is going to compute the core numbers, and then it's going to create and return a new graph that is the k-core of this graph.  Perhaps a copy/paste error.